README.md

```diff
@@ -4,111 +4,136 @@ An API for validating survey schemas.
 
 ## Setup
 
-It is recommended that you use [Pyenv](https://github.com/pyenv/pyenv) to manage your Python installations.
+In order to run locally you'll need Node.js, poetry and pyenv installed.
 
-### Install Poetry
+### Install NVM and pyenv
+
+NVM and pyenv will manage your versions of Node and Python.
+``` shell
+brew install nvm pyenv
+nvm use
 ```
-curl -sSL https://install.python-poetry.org | python3 - --version 2.1.2
-poetry install
+
+If you get a message in the command line after running `nvm use` that the version of node specified in the `.nvmrc` file isn't installed, just follow the commands to install it.
+
+e.g.
+``` shell
+nvm install v22.15.0
 ```
 
-## Running 
+### Install JS dependencies
 
-The `AJV_VALIDATOR_URL` defaults to `http://localhost:5002/validate`.
+``` shell
+npm install
+```
 
-You can override this by setting the `AJV_VALIDATOR_SCHEME` , `AJV_VALIDATOR_HOST`, and `AJV_VALIDATOR_PORT` environment variables.
+### Install Poetry and Python dependencies
 
-Alternatively, you can override the entire URL by setting the `AJV_VALIDATOR_URL` environment variable directly.
+``` shell
+curl -sSL https://install.python-poetry.org | python3 - --version 2.1.2
+poetry install
+```
 
+## Running locally
 
 To run the app:
-
-```
+``` shell
 make run
 ```
+
+Validator defaults to running on `http://localhost:5002/validate`.
+
+You can override this by setting the `AJV_VALIDATOR_SCHEME` , `AJV_VALIDATOR_HOST`, and `AJV_VALIDATOR_PORT` environment variables.
+The defaults for these are:
+- `AJV_VALIDATOR_SCHEME` = "http"
+- `AJV_VALIDATOR_HOST` = "localhost"
+- `AJV_VALIDATOR_PORT` = "5002"
+
+Alternatively, you can override the entire URL by setting the `AJV_VALIDATOR_URL` environment variable directly.
+
 If you want to run the app locally using multiple server workers set reload to "False" in the Uvicorn settings in api.py:
-```python
+``` python
 uvicorn.run("api:app", workers=20, port=5001, reload=False)
 ```
-The validator can be called directly in the browser using the "/validate" endpoint and the "url" parameter for the address where the schema is located (eg. GitHub Gist raw json). Only URLs from the ONSDigital organization are accepted.
-```
-http://localhost:5001/validate?url=...
-```
 
+### Running against a URL
 
-## Testing
+Once validator is running, it can be called directly in the browser using the "/validate" endpoint and the "url" parameter for the address where the schema is located (eg. GitHub Gist raw json).
```

Comment:
I would mention it's `localhost:5001/validate` by default. And give an example from line 73 here.